docs/guides/digital-assets/transfer-lsp7-digital-asset.md

```diff
@@ -41,8 +41,8 @@ npm install ethers @lukso/lsp-smart-contracts
 
 ### Step 1 - Setup imports and constants
 
-At this point you will need a private key in order to transfer some tokens as well as the `LSP7Mintable` _token contract address_ and the _address of the Universal Profile_ that has some tokens.
-We will import `LSP7Mintable`, `UniversalProfile`, `KeyManager` in order to get the _ABIs_ of the contracts that we will interact with.
+At this point you will need a private key, this EOA should be a controller address for the UP with proper [**permissions**](../../standards/universal-profile/lsp6-key-manager.md#permissions) in order to transfer some tokens as well as the `LSP7Mintable` _token contract address_ and the _address of the Universal Profile_ that has some tokens.
```

Comment:
Break it down in bullet points.

-----
at this point, you will need:

- the address + ABI of the LSP7DigitalAsset token contract
- the address + ABI of the Universal Profile
- the private key of the controller of your Universal Profile

Let's also not mention here about permission, but just put a link to the LSP6 page to explain what is a controller.